### DISCUSSION TITLE
Disable and reset password to users

### DISCUSSION THREAD

**Discussion by aexit-jgonzalesm** [padloc/padloc#685] [Q&A] [answered]
Hi there,

Please, I need to know how can I disable a user and if a user forgot his password, how can they or I reset it?

Regards,

JP

**Comment by aexit-jgonzalesm** [comment]
I'm installed self-hosted version.

**Comment by MaKleSoft** [comment, marked as answer]
Simply send them to the `/recover` route on the PWA. For instance, here is how the link looks like for the official web app: https://web.padloc.app/recover

Alternatively, they can also click on the ... button in the unlock/login screen and select "Forgot Password".

**Reply by aexit-jgonzalesm** [reply]
Hi,

In Login screen doesn't appears "..." button. In unlock screen does it.

![image](https://user-images.githubusercontent.com/42145156/219716266-26b03fd1-3207-4004-8747-3194d512f2b5.png)

**Comment by aexit-jgonzalesm** [comment]
One more thing. When I put new password a warning popup appears for confirmation to delete personal vaults?

**Reply by MaKleSoft** [reply]
Yes, resetting your master password will delete all data in your personal vault, as there is no way to recover the data with the encryption key lost.
